modules/openshift-developer-cli-push-workflow.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/odo-architecture.adoc
+
+[id="odo-push-workflow_{context}"]
+
+= `{odo-title} push` workflow
+In this section `{odo-title} push` workflow is described. {odo-title} push is deploying user code on an OpenShift cluster with all the necessary OpenShift resources.
```

Comment:
Initial sentence sounds off. I would word it like you did the beginning of the assembly: "This section describes..."